src/components/AssetScanner/stories/full.md

```diff
@@ -0,0 +1,75 @@
+# Asset Scanner
+
+<!-- STORY -->
+
+#### Description:
+
+Using for recognising assets name strings from photo.
+
+#### Usage requirements:
+
+Component depends on few third-part APIs, which is provided by Google
+Vision (to recognize text on picture). The user has to insert props
+to the component with a valid API key for Google Vision. Follow this
+guide to generate an api-key:
+
+**Google Vision API**
+
+> Before you can use the Cloud Vision API, you must enable it for your project and generate your API key, see here how – [Enable the Vision API](https://cloud.google.com/vision/docs/before-you-begin).   
+
+> Don't forget for setting up API key restrictions – [Using API Keys](https://cloud.google.com/docs/authentication/api-keys).  
+
+Google Vision API key could be pass to component via **ocrKey** prop or you can passURL address via **ocrUrl** prop in case of own backend realisation of Google Vision API call.  
+
+
+#### Usage:
+
+```typescript jsx
+import React from 'react';
+import { AssetScanner } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+  const onUnauthorized = (error: any): void => {};
+  return (
+    <AssetScanner
+      onUnauthorized={onUnauthorized}
+      ocrKey={'YOUR_GOOGLE_VISION_KEY'}
+    />
+    
+  );
+  
+}
+```
+
+#### Available props:
+
+##### Required:
+
+_No required props_
```

Comment:
Really no one?

Reply:
users can use scanner without providing any props, but they might not get expected behaviour without providing some of optional params. In storybook they can find description of all props with some examples